backend/app/services/im/manager.py

```
@@ -0,0 +1,431 @@
+            await self._send_error_message(
+                message,
+                team_id,
+                f"Failed to process your message: {str(e)}",
+            )
+
+    async def _create_or_append_task(
+        self,
+        session: IMSession,
+        content: str,
+        team_id: int,
+        im_user: Any,
```

Ruff (0.14.13)

Unused method argument: `im_user`

(ARG002)